### ISSUE TITLE
TOP 5 Cynefin

### ISSUE THREAD

**Issue by feststelltaste** [howtolearn-de/howtolearn-de.github.io#12]
On Twitter, Dan North asked how to learn Cynefin. There were plenty of answers which seem interesting for a TOP 5 article.

https://twitter.com/tastapod/status/1061254753764732929?s=19

**Comment by erdlet** [comment]
Maybe interesting, but we don't have the knowhow for that right? So if we write an article about it, we aren't able to give feedback on the articles / blogs /whatever.

**Comment by feststelltaste** [comment]
Agreed. But some experts answered this tweet, so we have some potential candidates for future articles that we might want to ask :-). Marked as "inquiry" as a reminder.
